relayer/create_ibc_channel.go

```diff
@@ -20,22 +20,12 @@ func (r *Relayer) CreateIBCChannel(override bool, logFileOption utils.CommandOpt
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	createClientsCmd := r.getCreateClientsCmd(override)
-	status := "Creating clients..."
-	fmt.Printf("💈 %s\n", status)
-	if err := r.WriteRelayerStatus(status); err != nil {
-		return ConnectionChannels{}, err
-	}
-	if err := utils.ExecBashCmd(createClientsCmd, logFileOption); err != nil {
-		return ConnectionChannels{}, err
-	}
-
 	//after successful update clients, keep running in the background
 	updateClientsCmd := r.GetUpdateClientsCmd()
 	utils.RunCommandEvery(ctx, updateClientsCmd.Path, updateClientsCmd.Args[1:], 20, utils.WithDiscardLogging())
```

Comment:
how it supposed to work when no clients created?

Reply:
it loops without success until clients are created. as the commands of clients creation are part of the link creation, and not their own separate command,  it becomes more complicated to only run it after client creation. agree it's not ideal and we could query the chains until clients are created but I don't think it's worth the effort currently as the con is simply letting it fail for a few seconds until the clients are created.

Reply:
yeah but will it create the "txs" to release the empty blocks?
IIRC we did it for that

Reply:
yea it's fine. the txs for releasing empty blocks are required only after the create clients which are not dependent in that.